test-integration/scores.coffee

```diff
@@ -10,17 +10,17 @@ TEACHER_USERNAME = 'teacher01'
 
 describe 'HS Student Scores', ->
 
-  beforeEach ->
+  @beforeEach ->
     @user = new User(@)
     @calendar = new CalendarHelper(@)
     @scores = new ScoresHelper(@)
     @courseSelect = new CourseSelect(@)
     @user.login(TEACHER_USERNAME)
-    @courseSelect.goToCourseByName('Physics I')
+    @courseSelect.goTo('PHYSICS')
     @calendar.goStudentScores()
 
-  afterEach ->
-    @user.goHome()
+  # afterEach ->
+  #   @user.goHome()
 
```

Comment:
I like the generic `goTo` vs the `goToCourseByName`, should these stray comments be removed?